pilz_control/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>pilz_control</name>
-  <version>0.5.1</version>
+  <version>0.4.3</version>
```

Comment:
We should keep the version number here (i.e. 0.5.1)

Reply:
Maybe it would help to add a versioning scheme explanation to the Readme?

0.4.x is currently for kinetic releases, 0.5.x for melodic as far as I remember.